src/Google/Service/Resource.php

```diff
@@ -193,7 +197,12 @@ public function call($name, $arguments, $expected_class = null)
         null,
         $postBody
     );
-    $httpRequest->setBaseComponent($this->client->getBasePath());
+
+    if ($this->rootUrl) {
```

Comment:
I think we'll need to bump to 1.2 if we're being semver-ish, as this is potentially a BC break - in the past we would also use the configured basepath if one, but now we will override a configured base path with the one out of the service. Realistically I'm not sure this is really going to hit anyone though - WDYT? We should bump the version number in Client.php regardless to something.